pkg/docker/containers-view.jsx

```diff
@@ -308,6 +309,50 @@ var ImageDetails = React.createClass({
     }
 });
 
+var ImageSecurity = React.createClass({
+    render: function () {
+        var info = this.props.info;
+        var text, rows;
+        var args = {
+            time: info.finishedTime.format('MMM Do'),
+            type: info.scanType,
+            count: info.vulnerabilities.length
+        };
+
+        if (info.successful === false) {
+            text = _('The scan from $time ($type) was not successful.');
+
+        } else if (info.vulnerabilities.length === 0) {
+            text = _('The scan from $time ($type) found no vulnerabilities.');
+
+        } else {
+            text = cockpit.ngettext('The scan from $time ($type) found one vulnerability:',
+                                    'The scan from $time ($type) found $count vulnerabilities:',
+                                    info.vulnerabilities.length);
+
+            var rows = info.vulnerabilities.map(function (vulnerability) {
```

Comment:
Hmmm, looks like you have declared the variable above. I would just remove the 'var ' while merging ...